app/assets/stylesheets/projects.scss

```diff
@@ -11,51 +12,132 @@ $block-height: 300px;
   @extend .small-centered;
 }
 
+/* start da css for da buttons */
+.demo-btn {
+  display: block;
+  border-radius: 5px;
+  padding: 12px 20px;
+  font-size: 22px;
+  color: #fff;
+  width: 75%;
+  margin: 48px auto;
+}
+
+.demo-btn:active {
+  transform: translate(0px, 5px);
+  -webkit-transform: translate(0px, 5px);
```

Comment:
I think for Firefox you have to use `-moz-webkit-transform` as well. Better, we can just use http://bourbon.io/docs/#transform which handles everything.